packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -4647,7 +4647,16 @@ export default class Meeting extends StatelessWebexPlugin {
         }
       );
 
-      if (!isRetry) {
+      // if this was the first attempt, let's do a retry
+      let shouldRetry = !isRetry;
+
+      if (CallDiagnosticUtils.isSdpOfferCreationError(error)) {
```

Comment:
Question: Would the SDP Offer Creation fail only if the codec is missing? Should we have this check specific to the Codec error? or is it that if the SDP creation failed once for any reason, it will always fail?

Reply:
@kesava codec could not be present if firefox does not install the h264 plugin extension

Reply:
@mkesavan13 in theory there could be other reasons why we fail to create the SDP offer, but in practise the missing h264 is the only one we see happening in prod. Anyway, whatever the reason, if it does fail, then on retry it will fail the same way, because we don't change anything that could affect the SDP offer creation when we do a retry, so there is no point retrying.

Reply:
Thanks @marcin-bazyl & @chrisadubois